charts/prefect-server/values.schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/schema#",
   "type": "object",
-  "additionalProperties": false,
+  "additionalProperties": true,
```

Comment:
For context this means that something like:
<img width="777" alt="Screenshot 2024-05-01 at 2 48 44 PM" src="https://github.com/PrefectHQ/prefect-helm/assets/12978862/39983074-a216-4e27-818a-a46146c8502a">

is great.  But something like:
<img width="908" alt="Screenshot 2024-05-01 at 2 49 21 PM" src="https://github.com/PrefectHQ/prefect-helm/assets/12978862/c4dc8f88-fdbd-470b-a924-aa5d5d9b4335">

is still not permitted which makes sense for these use cases.

Reply:
Yes exactly that would work perfectly for us